src/components/hand-controls.js

```diff
@@ -179,21 +185,23 @@ module.exports.Component = registerComponent('hand-controls', {
   },
 
   gestureAnimationMapping: {
-    'pointing': 'pointing',
-    'pistol': 'pistol',
-    'fist': 'press',
-    'touch': 'touch',
-    'thumb': 'thumb'
+    '': 'Open',
+    'pointing': 'Point',
```

Comment:
right, the default (no gesture) may still have a named animation associated (as it does here)